internal/pkg/mapper/template/replacevalues/replace_test.go

```diff
@@ -212,6 +212,16 @@ func TestValues_Replace(t *testing.T) {
 			Id:          `row-in-template`,
 		},
 	)
+	replacements.AddContentField(
+		model.ConfigKey{BranchId: 1, ComponentId: `foo.bar`, Id: `12`},
+		orderedmap.Key{orderedmap.MapStep("key1"), orderedmap.MapStep("key2")},
+		"new value in config",
+	)
+	replacements.AddContentField(
+		model.ConfigRowKey{BranchId: 1, ComponentId: `foo.bar`, ConfigId: `12`, Id: `56`},
+		orderedmap.Key{orderedmap.MapStep("key3"), orderedmap.MapStep("key4")},
+		"new value in row 56",
+	)
```

Comment:
Unit test, nahradime hodnotu v config a row.
V teste mame 2 rows, ktore maju zadane `key3.key4`, ale nahradzujeme to iba v 2 row, podla  zadaneho `ConfigRowKey`.